### PR TITLE
#163681927 Get partner location and add to Email info

### PR DESCRIPTION
**Pull request by asamisellah**
#### What does this PR do?
Gets partner location and from API and adds it to the email information.

#### Description of Task to be completed?
- Add partner email as part of email info
- Add catch statement to slack offboarding job
- Add `updateOrCreatePartner` method to offboarding jobs

#### How should this be manually tested?
1. On terminal, run command ...

#### What are the relevant pivotal tracker stories?
[#163681927](https://www.pivotaltracker.com/n/projects/2197045/stories/163681927)

### CODE REVIEW

File: server/jobs/offboarding/slack.js [hunk 1]
```diff
@@ -14,8 +14,12 @@ const { SLACK_AVAILABLE_DEVS_CHANNEL_ID } = process.env;
...
-  accessChannel(fellow.email, SLACK_AVAILABLE_DEVS_CHANNEL_ID, 'invite');
-  const { slackChannels: { general } } = await getPartnerRecord(partnerId);
-  accessChannel(fellow.email, general, 'kick');
+  try {
+    const { fellow, client_id: partnerId } = placement;
+    accessChannel(fellow.email, SLACK_AVAILABLE_DEVS_CHANNEL_ID, 'invite');
+    const { slackChannels: { general } } = await getPartnerRecord(partnerId);
+    accessChannel(fellow.email, general, 'kick');
+  } catch (error) {
+    console.log(error.message);
```
Comment (nzediegwu1) [inline server/jobs/offboarding/slack.js]: What should happen in the `catch` block?

File: server/jobs/helpers.js [hunk 2]
```diff
@@ -56,5 +56,6 @@ export async function executeEmailAutomation(emailFunctions, placement) {
     // write automation success to database
   } catch (error) {
     // write automation failure to database
+    console.log(error.message);
```
Comment (nzediegwu1) [inline server/jobs/helpers.js]: You can remove the `console.logs` after debugging

File: server/jobs/offboarding/slack.js (hunk 1 above)
Comment (asamisellah) [reply server/jobs/offboarding/slack.js]: Currently, I am logging the error

File: server/jobs/helpers.js (hunk 2 above)
Comment (asamisellah) [reply server/jobs/helpers.js]: @nzediegwu1 I have made the requested change. Thanks for the feedback

**Comment by nzediegwu1** [comment]
I think the concerns of this PR has been handled in the next PR: [#165083767](https://github.com/andela/bp-esa-backend/pull/101). You might consider closing it

**Pull request closed without merging**